packages/runtime/src/components/builtin/Slot/SlotRoot.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import { ReactNode, useContext } from "react"
+import { PropsContext } from "../../../next/components/MakeswiftSlot"
+
+type Props = {
+  children: ReactNode
+  showFallback: boolean
+}
+
+export const SlotRoot = (props: Props) => {
+  const ctx = useContext(PropsContext)
+
+  return (
+      <>
+        { props.showFallback ? props.children : ctx.fallback}
```

Comment:
Have we verified our implementation of `findDOMNode` properly handles finding the DOM now whens switching from fallback to children?

Reply:
I haven't had any issues switching back and forth between them repeatedly fwiw

Reply:
with telegraph or otherwise

Reply:
Great to hear!